documentation/prometheus-mixin/Makefile

```diff
@@ -1,19 +1,28 @@
 JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
 
+include ../../Makefile.common
```

Comment:
This might not be correct - the top-level `Makefile.common` is for Prometheus components and not designed for mixins which have their own home at https://monitoring.mixins.dev/.